src/widgets/splits/SplitHeader.hpp

```diff
@@ -85,7 +85,6 @@ class SplitHeader final : public BaseWidget, pajlada::Signals::SignalHolder
     std::vector<pajlada::Signals::ScopedConnection> channelConnections_;
 
 public slots:
-    void moveSplit();
```

Comment:
Just because of your other comment :upside_down_face: :tf:

Reply:
:tf: